clients/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -38,21 +41,30 @@ import {
   useGetAllDataUsesQuery,
   useUpdateDataUseMutation,
 } from "../data-use/data-use.slice";
+import { parentKeyFromFidesKey } from "./helpers";
 import {
   useCreateDataCategoryMutation,
   useDeleteDataCategoryMutation,
   useGetAllDataCategoriesQuery,
   useUpdateDataCategoryMutation,
 } from "./taxonomy.slice";
-import type { FormValues } from "./TaxonomyFormBase";
-import { Labels, TaxonomyEntity } from "./types";
+import { FormValues, Labels, TaxonomyEntity } from "./types";
 
 export interface TaxonomyHookData<T extends TaxonomyEntity> {
   data?: TaxonomyEntity[];
   isLoading: boolean;
   labels: Labels;
-  handleCreate: (entity: T) => RTKResult<TaxonomyEntity>;
-  handleEdit: (entity: T) => RTKResult<TaxonomyEntity>;
+  resourceType?: ResourceTypes;
```

Comment:
I added this so that components that use `TaxonomyHookData` could know what resource they're working with. However, it didn't turn out to be that useful outside of the body of the hook (so far).